### ISSUE TITLE
Can I use dbhub.io with SQLAlchemy?

### ISSUE THREAD

**Issue by swckd** [sqlitebrowser/dbhub.io#372]
Can I connect my SQLAlchemy to dbhub.io?
app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///kanban.db'

**Comment by chrisjlocke** [comment]
Don't think it would work directly .. dbhub.io works via API, not a direct 'storage'.  But you could download the file locally and do what you needed, then upload again?
There are live databases on dbhub.io, but again, they work on an API, not a direct storage.

**Comment by swckd** [comment]
It makes sense! Thanks!

**Comment by justinclift** [comment]
> dbhub.io works via API

That's correct.  The DBHub.io service doesn't have a Python driver written, so SQLAlchemy doesn't know how to communicate with it.

Maybe some day... :smile:
